### PR TITLE
added bubble sort algorithm

### PR DESCRIPTION
**Pull request by A713F3**
added iterative and recursive bubble sort algorithm

### CODE REVIEW

**Comment by SatinWukerORIG** [comment]
Thank you for your contribution! My apology that I was unable to maintain my GitHub for the past 2 weeks due to my busy job hahaha. I just added a [contribution guideline](/CONTRIBUTING.md), and don't worry, I will give you feedback

File: sorts/bubble_sort.f90 [hunk 1]
```diff
@@ -0,0 +1,44 @@
+program bubble_sort_program
```
Comment (SatinWukerORIG) [inline sorts/bubble_sort.f90:1]: Please some doc strings before the program to introduce this program to learners with some basic info, such as time and space complexity, and include a link! 👍

File: sorts/bubble_sort.f90 [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    call bubble_sort(array)
+
+    print*, "After:", array
+
+contains
+    
+    subroutine bubble_sort(collection)
+        real, dimension(:), intent(inout) :: collection
+
+        integer :: i, j, collection_size
```
Comment (SatinWukerORIG) [inline sorts/bubble_sort.f90:19]: Please also add comments throughout these programs! Here, you swap the values of 2 variables using a temporary variable, therefore you can add a comment explaining this here 👍🥇

**Comment by A713F3** [comment]
> Thank you for your contribution! My apology that I was unable to maintain my GitHub for the past 2 weeks due to my busy job hahaha. I just added a [contribution guideline](/CONTRIBUTING.md), and don't worry, I will give you feedback

Ok thanks 👍🏻. I'm going to check the guidelines and refactor the commit if necessary. That's my first ever contribution so there are probably problems lol.

**Review by SatinWukerORIG — APPROVED** [review]
Anyways I am okay with merging it. I will add algorithm descriptions and other comments, don't worry haha!

**Comment by A713F3** [comment]
Okay, thank you so much. I wasn't sure how to refactor it so I was waiting for some examples to be committed. I will add comments for my contributions as shown in example programs 👍.

**Pull request merged**